src/main/java/com/banking/bankservice/web/BankAccountController.java

```diff
@@ -16,9 +19,44 @@
     private RestTemplate restTemplate;
 
     @GetMapping
-    public Account get() {
-        ResponseEntity<Account> response = restTemplate.getForEntity("http://localhost:8090/account", Account.class);
+    public Account getAll() {
+        ResponseEntity<Account> response = restTemplate.getForEntity("http://localhost:8085/account", Account.class);
```

Comment:
The string `http://localhost:8085/account` is duplicated 7 times! What would clean code say? :P

Reply:
Done at last commit